WalletWasabi.Fluent/ViewModels/CoinControl/SelectCoinsDialogViewModel.cs

```diff
@@ -0,0 +1,54 @@
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using Avalonia.Controls;
+using WalletWasabi.Blockchain.TransactionOutputs;
+using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Fluent.Models;
+using WalletWasabi.Fluent.ViewModels.CoinControl.Core;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+using WalletWasabi.Fluent.ViewModels.Wallets;
+using WalletWasabi.Fluent.ViewModels.Wallets.Send;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinControl;
+
+[NavigationMetaData(
+	Title = "Select Coins",
+	Caption = "",
+	IconName = "wallet_action_send",
+	NavBarPosition = NavBarPosition.None,
+	Searchable = false,
+	NavigationTarget = NavigationTarget.DialogScreen)]
+public partial class SelectCoinsDialogViewModel : DialogViewModelBase<IEnumerable<SmartCoin>>
+{
+	public SelectCoinsDialogViewModel(WalletViewModel walletViewModel, TransactionInfo info, IEnumerable<SmartCoin> transactionSpentCoins)
```

Comment:
info and transactionSpentCoins is unused

Reply:
I thought it was a good idea to leave there the parameters that will be used in Stage 1, since this is an intermediate PR.
Removed anyways 😁